pkg/aspect/quotasManager.go

```diff
@@ -117,21 +117,19 @@ func (*quotasManager) ValidateConfig(config.AspectParams, expr.Validator, descri
 	return
 }
 
-func (w *quotasWrapper) Execute(attrs attribute.Bag, mapper expr.Evaluator, ma APIMethodArgs) Output {
-	qma := ma.(*QuotaMethodArgs)
-
+func (w *quotasWrapper) Execute(attrs attribute.Bag, mapper expr.Evaluator, qma *QuotaMethodArgs) (rpc.Status, *QuotaMethodResp) {
 	info, ok := w.metadata[qma.Quota]
 	if !ok {
 		msg := fmt.Sprintf("Unknown quota '%s' requested", qma.Quota)
 		glog.Error(msg)
-		return Output{Status: status.WithInvalidArgument(msg)}
+		return status.WithInvalidArgument(msg), nil
 	}
 
 	labels, err := evalAll(info.labels, attrs, mapper)
 	if err != nil {
 		msg := fmt.Sprintf("Unable to evaluate labels for quota '%s' with err: %s", qma.Quota, err)
```

Comment:
as above: any place where the error could be a multi-error, I think we should try to use DebugInfo, etc. and have more user-friendly messages.